mappings/net/minecraft/item/SpawnEggItem.mapping

```diff
@@ -15,6 +15,6 @@ CLASS bcz net/minecraft/item/SpawnEggItem
 	METHOD a isOfSameEntityType (Lib;Laim;)Z
 		ARG 1 tag
 		ARG 2 type
-	METHOD b entityTypeFromTag (Lib;)Laim;
+	METHOD b getEntityType (Lib;)Laim;
 		ARG 1 tag
-	METHOD d iterator ()Ljava/lang/Iterable;
+	METHOD d getSpawnEggs ()Ljava/lang/Iterable;
```

Comment:
Much needed change. Although it is a static method, so it could be made simpler. Would something like `SpawnEggItem.getAll()` work ?

Reply:
Resolved by cc5abf1